a3fe/tests/README.md

```diff
@@ -6,23 +6,26 @@ This directory contains integration tests for A3FE, which are designed to run in
 
 To run integration tests, use the following command:
 
-```bash
-cd your/path/a3fe
-RUN_SLURM_TESTS=1 pytest a3fe/tests --run-integration -v
-```
+   ```bash
+   cd your/path/a3fe
+   RUN_SLURM_TESTS=1 pytest a3fe/tests --run-integration -v
+   ```
+Or to run a specific integration test:
+   ```bash
+   RUN_SLURM_TESTS=1 pytest a3fe/tests/test_run_integration.py::TestSlurmIntegration::test_slurm_calculation_setup --run-integration -v
+   ```
 
 ## Skipping Integration Tests
 
 Integration tests are skipped by default. There are several ways to control which tests run:
 
-1. To skip all integration tests (default behavior):
+   To skip all integration tests (default behavior):
    ```bash
    pytest a3fe/tests
    ```
-
-2. To run a specific integration test:
+   Or to explicitly skip integration tests (same as deafult):
    ```bash
-   RUN_SLURM_TESTS=1 pytest a3fe/tests/test_run_integration.py::TestSlurmIntegration::test_slurm_calculation_setup --run-integration -v
+   RUN_SLUEM_TESTS=0 pytest a3fe/tests --run-integration -v
```

Comment:
Small typo "SLUEM". Also, should --run-integration be removed to avoid confusion? My understanding is that this says "run the integration tests" but RUN_SLURM_TESTS=0 stops them running. In the future, these commands might do different things.